pkg/rollout/rollout.go

```diff
@@ -318,6 +364,23 @@ func (r *Rollout) diagnoseCandidate(candidate string, healthCriteria []config.Me
 	return d, errors.Wrap(err, "failed to diagnose candidate's health")
 }
 
+// shouldRollForward determines if enough time has passed since last roll
+// forward.
+func (r *Rollout) shouldRollForward(lastRollForwardStr string, timeBetweenRollForwards int) (bool, error) {
```

Comment:
`shouldRollForward` → `isTimeElapsed`

Reply:
I renamed it to `hasEnoughTimeElapsed`. Is it fine?